pkg/cli/secret_encrypt.go

```diff
@@ -0,0 +1,63 @@
+package cli
+
+import (
+	cli "github.com/acorn-io/acorn/pkg/cli/builder"
+	"github.com/acorn-io/acorn/pkg/cli/builder/table"
+	"github.com/acorn-io/acorn/pkg/client"
+	"github.com/acorn-io/acorn/pkg/encryption/nacl"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func NewSecretEncrypt() *cobra.Command {
+	cmd := cli.Command(&Encrypt{}, cobra.Command{
+		Use:          "encrypt [flags] STRING",
+		SilenceUsage: true,
+		Short:        "Encrypt string information with clusters public key",
+		Args:         cobra.RangeArgs(1, 1),
+	})
+	return cmd
+}
+
+type Encrypt struct {
+	PublicKey []string `usage:"Pass one or more cluster publicKey values"`
+}
+
+func (e *Encrypt) Run(cmd *cobra.Command, args []string) error {
+	out := table.NewWriter([][]string{
+		{"Name", "{{.}}"},
+	}, "", true, "")
+
+	c, err := client.Default()
+	if err != nil {
+		return err
+	}
+
+	if len(args[0]) > 4096 {
```

Comment:
is this our arbitrary limit or does something break if its bigger than that?

Reply:
It's the limit of libsodium. GH and golang mention it.